utility/src/main/java/org/oppia/util/parser/LiTagHandler.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.util.parser
+
+import android.text.Editable
+import android.text.Html
+import android.text.Spannable
+import android.text.Spanned
+import android.text.style.BulletSpan
+import org.xml.sax.XMLReader
+
+/**
+ * [Html.TagHandler] implementation that processes <li> tags and creates bullets.
+ *
+ * Reference: https://github.com/davidbilik/bullet-span-sample
+ */
+class LiTagHandler : Html.TagHandler {
```

Comment:
Please add tests for this class (preferably via HtmlParser's tests) to verify that the custom span is added as expected.

Reply:
Added test cases